portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase

```diff
@@ -9,7 +9,7 @@ definition {
 	setUp {
 		TestCase.setUpPortalInstance();
 
-		User.firstLoginUI();
+		User.firstLoginPG();
```

Comment:
@Diogomes, is this second commit necessary? Can you please clarify?

Reply:
I made the adjustment through github web